build/resources/main/data/player.json

```diff
@@ -0,0 +1,54 @@
+{
+  "type": "Player",
+  "name": "you",
+  "description": "You're just a regular baseline human.",
```

Comment:
Regular & baseline is kind of the same. Maybe change the description into something more flavorful? A generic human can still be interesting if you can write it that way. example: "You're the average Joe. If only there existed some Joe's to take the average from."